logger/scenario_logger_msgs/msg/Log.msg

```diff
@@ -1,6 +1,6 @@
-duration elapsed_time
+builtin_interfaces/Duration elapsed_time
 scenario_logger_msgs/Level level
 string[] categories
 # description in json format
 string description
-string from 
\ No newline at end of file
+string source
```

Comment:
This is changed from `from` to `source` since following error showed up during colcon build:

```
Member name 'from' in the message 'Log' is a reserved keyword in Python and is not supported at the moment. Please use a different name.
```